src/renderer/components/NewProjectModal/ImportMediaView.tsx

```diff
@@ -52,51 +46,35 @@ const ImportMediaView = ({ prevView, closeModal, nextView }: Props) => {
     (store: ApplicationStore) => store.currentProject
   );
 
+  const dispatch = useDispatch();
+
+  const setCurrentProject = (project: RuntimeProject) => {
+    return dispatch(projectCreated(project));
+  };
+
   // Reset the import - for when delete button is pressed on media
   const removeMediaFromImport: () => void = () => {
     setIsAwaitingMedia(true);
     setMediaFilePath(null);
     setMediaFileName(null);
   };
 
-  const dispatch = useDispatch();
-
   if (currentProject === null) {
     return null;
   }
 
   const projectName = currentProject.name;
 
-  const setCurrentProject = (project: RuntimeProject) =>
-    dispatch(projectCreated(project));
-
   const handleTranscribe = async () => {
-    if (mediaFilePath === null) {
-      return;
-    }
-
-    const projectWithMedia = await updateProjectWithMedia(
+    const projectWithMedia = await updateProjectMediaFilePath(
```

Comment:
This whole `updateProjectMediaFilePath` util probably isn't necessary, you can just do

```
const projectWithMedia = {
   ...currentProject,
   mediaFilePath,
};
```

And then check if mediaFilePath was null for the early exit